void_migration/void_migration.py

```diff
@@ -581,15 +820,27 @@ def time_march(p):
             u, v, s = close_voids(u, v, s)
 
         if t % p.save_inc == 0:
-            plotter.make_saves(x, y, s, u, v, c, T, p, t)
+            if hasattr(p,"charge_discharge"):
```

Comment:
can you move this logic into the make_saves function instead to keep the time_march function as clean as possible?